marketplace/src/main/resources/install/templates/retention/nxserver/config/retention-compliance-config.xml.nxftl

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.retention.compliance.types">
+<#if "${nuxeo.retention.compliance.enabled}" == "true">
+    <require>org.nuxeo.retention.types</require>
+
+    <extension target="org.nuxeo.ecm.core.schema.TypeService" point="schema">
+        <schema name="files" enabled="false" />
+    </extension>
+
+    <extension target="org.nuxeo.ecm.core.schema.TypeService" point="doctype">
+        <facet name="Commentable" enabled="false" />
+        <facet name="Publishable" enabled="false" />
+        <facet name="Versionable" enabled="false" />
+    </extension>
+</#if>
```

Comment:
We use 2 spaces indentation for XML files (missed it also on `master` 😞).

Reply:
Already merged on master, but I can open a new PR if you think it is better. 😞